deptofdefense/move.mil
# Add homepage content

## Checklist

I have…


## Summary of Changes

This pull request reincorporates the homepage buildout in #12.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout add-homepage-content`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. `bin/rails server`, and
1. load up http://localhost:3000 in your Web browser of choice.

## Screenshots

### Narrow viewport

![narrow](https://user-images.githubusercontent.com/27780860/28894854-29f370b8-77a4-11e7-871e-129d53ce9079.png)

### Wide viewport

![wide](https://user-images.githubusercontent.com/27780860/28894857-2dcd4e98-77a4-11e7-894d-c0503efa26a3.png)
